test/e2e/wireguard_test.go

```diff
@@ -40,6 +40,7 @@ func TestWireGuard(t *testing.T) {
 		t.Fatalf("Error when setting up test: %v", err)
 	}
 	defer teardownTest(t, data)
```

Comment:
We used to add mutate modification on the config, see line 52-53. If here we skip the case, do you want to revert the mutate changes? In this way, I'm afraid the case may be skip in the kind job which enable all features.

Reply:
It's Ok to keep the old mutate change, I can remove my change in wireguard_test.go since existing codes will make the configuration right before the test is running.